backend/multi/backend.c

```diff
@@ -116,6 +116,9 @@ void wlr_multi_backend_add(struct wlr_backend *multi,
 	wl_signal_add(&backend->events.output_remove, &sub->output_remove);
 
 	list_add(multi->state->backends, sub);
+
+	if(backend->egl)
+		multi->egl = backend->egl;
```

Comment:
```c
if (backend->egl) {
\tmulti->egl = backend->egl;
}
```

cc @ascent12: this'll probably throw a bit of a wrench into your multi-GPU plans